cooked-validators/src/Cooked/MockChain/Ltl.hs

```diff
@@ -0,0 +1,285 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+module Cooked.MockChain.Ltl where
```

Comment:
It seems that this module has nothing to do with `MockChain`; I'd suggest we move it to `Cooked.Ltl`